lib/elixir/src/elixir_tokenizer.erl

```diff
@@ -955,6 +955,12 @@ handle_space_sensitive_tokens(String, Line, Column, Scope, Tokens) ->
 
 %% Helpers
 
+blue(Message) ->
+  case application:get_env(elixir, ansi_enabled, false) of
+    true -> ["\e[34m", Message, "\e[0m"];
+    false -> Message
+  end.
```

Comment:
Maybe we should move this to :elixir_errors, so we can reuse it here and in other hints. Perhaps we can call it `hint` or something. :)

Reply:
I've used the existing infrastructure we had in the `prefix` function, added the hint case and exposed it, so it can also be used for `warning:` and `error:` in the future in other places

I've also updated the other tests we had to assert on the changes from HINT to the ansi hint.
I'm not sure if it's the best approach, but since tests run asynchronously I think we're better just asserting on the ANSI characters directly rather than trying to deactivate it